README.md

```diff
@@ -95,6 +97,30 @@ If you do not have a working Golang environment, please follow [How to install G
 go get -u github.com/minio/mc
 ```
 
+## Configure
+
+By default `mc` is configured to interact with the _play_ environment, a public Minio server. If you want to configure a different server:
+
+```sh
+mc config host add <ALIAS> <YOUR-S3-ENDPOINT> <YOUR-ACCESS-KEY> <YOUR-SECRET-KEY>
+```
+
+You will be able to list buckets (i.e. `mc ls <ALIAS>`) or any other available operation.
+
+### Endpoint and credentials
+
+At startup, Minio server provides its endpoint, access key and secret key.
```

Comment:
We don't need to talk about `Endpoint and credentials` from just Minio server point of view. `mc` is a generic tool to be used with all S3 compatible servers,  we can move this to `# Example - Minio Cloud Storage` section.